### PR TITLE
Separate out ChatClass into Services

### PR DESCRIPTION
**Pull request by ayc19**
Moving chatClass internally to 3 different things: db, tables, and queries

1. db only creates/closes the db
2. tables creates/drops the tables
3. queries does the queries

Was thinking about it like a dependency graph of like

- db
- tables (db)
- queries (db, tables

### CODE REVIEW

File: electron/app/index.tsx [hunk 1]
```diff
@@ -3,9 +3,11 @@ import { render } from 'react-dom';
 import { AppContainer as ReactHotAppContainer } from 'react-hot-loader';
 import './app.global.css';
 import * as sqlite3 from 'sqlite3';
-import log from 'electron-log';
```
Comment (alexdanilowicz) [inline electron/app/index.tsx:6]: why the removal of electron-log here? Oh is it in order for you to more easily debug in the window?

File: electron/app/chatBro/index.ts [hunk 2]
```diff
@@ -1,24 +1,3 @@
...
-  );
-  return settleAll(dropTablePromises);
-}
-
-const tables = ChatBro.Tables;
-
-export { dropAllExistingTables, createAllTables, getWordCount, tables };
+export * from './db'
+export * from './tables'
+export * from './queries'
```
Comment (ayc19) [inline electron/app/chatBro/index.ts:3]: basically moving out chat bro stuff internally to 3 different things, db, tables, and queries

db only creates/closes the db

tables creates/drops the tables

queries does the queries

Was thinking about it like a dependency graph of like

1. db
2. tables (db)
3. queries (db, tables)

File: electron/app/chatBro/db/index.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import * as _ from 'lodash';
+import * as sqlite3 from 'sqlite3';
+
+export function initializeDB(): sqlite3.Database {
+  const sqldb = sqlite3.verbose();
+  const db = new sqldb.Database(`${process.env.HOME}/Desktop/chat.db`);
```
Comment (alexdanilowicz) [inline electron/app/chatBro/db/index.ts:6]: `${process.env.HOME}/Desktop/chat.db` should read from the constants file I have set up ... going to add a commit

File: electron/app/chatBro/index.ts (hunk 2 above)
Comment (alexdanilowicz) [reply electron/app/chatBro/index.ts:3]: This is sick. Wanna add what you wrote here to the README?

File: electron/app/index.tsx (hunk 1 above)
Comment (alexdanilowicz) [reply electron/app/index.tsx:6]: We should use electron-log for everything. Including things we want to see in the console. See: https://www.npmjs.com/package/electron-log#transport

**Pull request merged**